flink-formats/flink-orc/src/test/java/org/apache/flink/orc/OrcFileSystemITCase.java

```diff
@@ -208,6 +208,15 @@ void testOrcFilterPushDown() throws ExecutionException, InterruptedException {
                 Collections.singletonList(Row.of("x10", "10")));
     }
 
+    @TestTemplate
+    void testOrcFilterPushDownLiteralFirst() throws ExecutionException, InterruptedException {
+        super.tableEnv()
+                .executeSql("insert into orcLimitTable values('a', 10, 10), ('b', 11, 11)")
+                .await();
+        check("select y from orcLimitTable where 10 >= y", Collections.singletonList(Row.of(10)));
```

Comment:
Could you please add the test that reflect your changes in `OrcFilters.java` here (e.g., with `>`, '<') ?